pandas/core/indexes/datetimes.py

```diff
@@ -2241,21 +2241,56 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
     -------
     rng : DatetimeIndex
 
+    Examples
+    --------
+
+    Given the ``start`` and ``end`` mandatory parameters,
+    with default `freq='D'` (Days)
+
+    >>> pd1 = pd.date_range(start='1/1/2018', end='1/10/2018')
```

Comment:
As the output is not being used more than to be displayed, I'd show it directly instead of assigning it to a variable. If you need to save any of them `idx` is more standard than `pd1`.